ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Node/useNodeSummaryData.tsx

```diff
@@ -1,8 +1,9 @@
 import { gql, useQuery } from '@apollo/client';
+
 import {
     ResourceCountByCveSeverityAndStatus,
     resourceCountByCveSeverityAndStatusFragment,
-} from 'Containers/Vulnerabilities/WorkloadCves/SummaryCards/CvesByStatusSummaryCard';
```

Comment:
Oh, interesting! This is counterpart to new `'no-relative-path-to-src-in-import'` rule that what could be relative path within container has pseudo-absolute path. It also explains why only one error for the new rule, because this is the situation that was in the back of my mind when I wrote it.

What do you think about pro and con of a rule to enforce relative path? Link what you wrote, leaving aside that you had moved the file.